main/src/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,199 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    int water = 400;
```

Comment:
@Edvard777 why not private ?

Reply:
If about Class modifier, if private i can't call it from package.Can make Default modifier